tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -382,98 +382,139 @@ def search_versions(filter_string):
         assert mvds[0].source == "A/B"
         assert mvds[0].description == "Online prediction model!"
 
+    def search_registered_models(self,
+                                 filter_string,
+                                 page_token=None,
+                                 max_results=10):
+        return self.store.search_registered_models(filter_string, page_token, max_results)
+
+    def get_rm_names(self, rms):
+        return [rm.name for rm in rms]
```

Comment:
looks like these 2 methods are always used together. would it make sense to always convert registered models -> names and token from this `_search_registered_models` method. I would make it the `_` private method prefix